packages/tools/storybook/stories/components/accordion/accordion.web-component.stories.page.mdx

```diff
@@ -1,7 +1,7 @@
 import { Canvas, Description, Meta, DocsContainer } from '@storybook/addon-docs';
 
-import Usage from '@ovhcloud/ods-stencil-accordion/docs/osds-accordion/usage.mdx';
-import APITable from '@ovhcloud/ods-stencil-accordion/dist/docs/components/osds-accordion/readme.md';
+import Usage from '@ovhcloud/ods-component-accordion/documentation/osds-accordion/usage.mdx';
```

Comment:
still generated in docs for now, will be moved later

Reply:
i think it's not auto generate